osm/src/main/groovy/org/orbisgis/osm/Loader.groovy

```diff
@@ -125,59 +116,56 @@ IProcess fromPlace() {
         title "Extract the OSM data using a place name"
         inputs datasource: JdbcDataSource, placeName: String, distance : 0
         outputs zoneTableName: String, zoneEnvelopeTableName: String, osmTablesPrefix: String, epsg:int
-        run { datasource,placeName,distance ->
-            String formatedPlaceName = placeName.trim().split("\\s*(,|\\s)\\s*").join("_");
-            def outputZoneTable = "ZONE_${formatedPlaceName}_${UUID.randomUUID().toString().replaceAll("-", "_")}"
-            def outputZoneEnvelopeTable = "ZONE_ENVELOPE_${formatedPlaceName}_${UUID.randomUUID().toString().replaceAll("-", "_")}"
-            def osmTablesPrefix = "OSM_DATA_${formatedPlaceName}_${UUID.randomUUID().toString().replaceAll("-", "_")}"
+        run { JdbcDataSource datasource, placeName, distance ->
```

Comment:
JdbcDataSource not necessary

Reply:
Right, but without, the IDE doesn't detect well the completion, do I remove it ?